recipes/acado/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "1.2.2-beta":
```

Comment:
beta and alpha versions are not allowed in CCI. The interesting thing is, there is no stable versions, only prereleases: https://github.com/acado/acado/tags

Reply:
Here I would say that is better to have a `1.2.2-beta` than a `cci.<date>`, at least it matches a released version... it looks like there is not going to be a stable release (in the near future).

Reply:
Indeed. I'll open an issue to the author an ask about.

Reply:
Don't get your hopes high, there was no development for years.

Reply:
@blackliner thanks for the information. Well, in that we can use that beta version.